src/test/java/com/ot/restaurant/repositories/CustomerRepositoryImplTest.java

```diff
@@ -0,0 +1,89 @@
+package com.ot.restaurant.repositories;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.ot.restaurant.entities.Customer;
+import com.ot.restaurant.exceptions.IdNullException;
+import constants.Status;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CustomerRepositoryImplTest {
+  private CustomerRepositoryImpl customerRepository = new CustomerRepositoryImpl();
+  private List<Customer> customerListTest = new ArrayList<>();
+  private final String firstName = "Loki1";
+  private final String secondName = "Loki2";
+  private final String thirdName = "Loki3";
+
+  private Customer createCustomer(String firstName) {
+    Customer customer = new Customer();
+    customer.setFirstName(firstName);
+    customer.setStatus(Status.ACTIVE);
+    return customer;
+  }
```

Comment:
Create a new class `CustomerFixtures` within a fixtures folder. There you should be able to build any kind of customer with any kind of parameters.

You should be able to use this way:

```
final var customer = CustumerFixtures.buildDefaultCustomer();
or
final var customerExample = Customer.builder().withId(1).withName("test").build();
final var customer = CustumerFixtures.buildCustomerFromExample(customerExample );
```